hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -142,6 +142,13 @@ def lgt_to_gt(lgt, la):
 
 
 def summarize(mt):
+    """Computes summary statistics
+
+    Note
+    ----
+    You will not be able to run `combine_gvcfs` with the output of this
```

Comment:
Sorry. Just saw that `combine_gvcfs` is not linked to the function. i.e. `:func:...`. See other instances below.